scss/components/_buttons.scss

```
@@ -406,4 +406,10 @@ $btn-border-width: 4px;
       box-shadow: none;
     }
   }
+}
+
+@media screen and (-ms-high-contrast: active) {
+  .btn-primary {
```

This should apply to all buttons, not just `btn-primary`.
```suggestion
  .btn {
```

---

Addressed